tests/utls/context-mock.ts

```diff
@@ -0,0 +1,12 @@
+import {contextService} from "../../src/services/context.service";
+
+export function setDefaultProfile(): void {
```

Comment:
Is this needed on a global level (should all tests have this by default)?

Reply:
Apart from tests for profile commands, I think all tests will need this.